doc/api/fs.md

```diff
@@ -332,6 +332,33 @@ If one or more `filehandle.read()` calls are made on a file handle and then a
 position till the end of the file. It doesn't always read from the beginning
 of the file.
 
+#### `filehandle.readJSON(options)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object|string}
+  * `encoding` {string|null} **Default:** `null`
```

Comment:
I don't think `null` makes a lot of sense as a default for JSON parsing, it's effectively `utf8`.
```suggestion
  * `encoding` {string|null} **Default:** `utf8`
```